settings/kickofftimerplugin.set

```diff
@@ -1,6 +1,13 @@
 Kickoff timer plugin
+9|Controls
+8
 0|Load|plugin load kickofftimerplugin
 7
 0|Unload|plugin unload kickofftimerplugin
 7
 0|Reset Personal Best|kickofftimer_reset
+7
+0|Reset Personal Best and Delete file|kickofftimer_resetAndDelete

```

Comment:
I dont like the delete thing. Nobody cares about a single file getting deleted or not. Just overhead.

Reply:
I've removed this button and the functions from it